.travis/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e -u
+
+py38="$(python -c 'import sys; print(sys.version_info >= (3, 8))')"
+if [ "$py38" = "True" ]; then
+    # Python 3.8 deprecates some features still in use to support Python 3.5
+    # or are used by dependencies, so we can't yet fail on
+    # DeprecationWarnings.
+    warn_args=""
+else
+    warn_args="
+        -Werror
+        -Wignore::DeprecationWarning:site
+        -Wignore::PendingDeprecationWarning:nose.importer
+        -Wignore::DeprecationWarning:nose.importer
+        -Wignore::DeprecationWarning:nose.suite"
+fi
+
+set -x
+python $warn_args `which nosetests` --with-coverage --cover-erase --cover-package aiokatcp
```

Comment:
Running [shellcheck](https://github.com/koalaman/shellcheck) on the script reports.

```bash 
In run.sh line 20:
python $warn_args `which nosetests` --with-coverage --cover-erase --cover-package aiokatcp
       ^-- SC2086: Double quote to prevent globbing and word splitting.
                  ^-- SC2046: Quote this to prevent word splitting.
                  ^-- SC2006: Use $(..) instead of legacy `..`.
```

Reply:
I *want* the word splitting...